mandatory/1-fix-functions.js

```diff
@@ -22,31 +21,26 @@ function mood() {
 
 function greaterThan10(num) {
```

Comment:
The argument `num` that is passed into your function is the number that you are checking. You should define isBigEnough to indicate if that number is greater than 10.